src/Tracking/README.md

```diff
@@ -149,7 +149,8 @@ All event names are prefixed by `wcadmin_gla_`.
 
 -   `launch_paid_campaign_button_click` - Triggered when the "Launch paid campaign" button is clicked to add a new paid campaign
 
-    -   `audience`: country code of the paid campaign audience country
+    -   `audience`: country code of the paid campaign audience country. This is inapplicable when a campaign is created with the multi-country targeting feature.
```

Comment:
❓  We have actually this possibility? I mean to have only `audience`

Reply:
After introducing the multi-country targeting feature, the `audience` property will no longer appear in the tracking data. I kept it as a record so that when looking at older tracking data, this property can still have a reference. Otherwise, there's no other document about it.

Reply:
Oh, gotcha. Is this how we do this normally? Wondering if would be better to just have it 

`audiences : country codes of the paid campaign audience countries, e.g. 'US,JP,AU'. Before multi-country support, it was implemented as 'audience'`

or even better IMO

`audience : country codes of the paid campaign audience countries, e.g. US,JP,AU'`

Maintaining the same name. Since "Audience" in the singular form also evokes a group, it makes sense also in naming.

Reply:
WDYT?

Reply:
> After introducing the multi-country targeting feature, the `audience` property will no longer appear in the tracking data.

Sorry I'm not quite right here. Even after the feature is released, there will still be merchants who are staying with old versions or haven't updated it yet, so the `audience` property will still be in the tracking data for quite some time.

> or even better IMO
> 
> `audience : country codes of the paid campaign audience countries, e.g. US,JP,AU'`
> 
> Maintaining the same name. Since "Audience" in the singular form also evokes a group, it makes sense also in naming.

Using the same property name will result in the following two types of data being identical, and it will be hard to identify which campaign is which:
- The old single-country targeting selected `US`
- The new multi-country targeting also selects only one country `US`

I updated their descriptions in https://github.com/woocommerce/google-listings-and-ads/pull/1273/commits/898c016f7e4dfd871c6192a2b5f641e4f5aac811, hope it could be more clear.